data/saas/config/firebase_auth_config.yml

```diff
@@ -0,0 +1,54 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Firebase Auth SaaS Config
+  type: firebase_auth
+  description: A sample schema representing the Firebase Auth connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+      default_value: https://identitytoolkit.googleapis.com
+    - name: type
+      default_value: service_account
+    - name: project_id
+    - name: private_key_id
+    - name: private_key
+    - name: client_email
+    - name: client_id
+    - name: auth_uri
+      default_value: https://accounts.google.com/o/oauth2/auth
+    - name: token_uri
+      default_value: https://oauth2.googleapis.com/token
+    - name: auth_provider_x509_cert_url
+      default_value: https://www.googleapis.com/oauth2/v1/certs
+    - name: client_x509_cert_url
+
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: api_key
+      configuration:
+        query_param:
```

Comment:
Should this be `query_params`?

Reply:
good catch! i didn't mean to commit this, it was from an earlier iteration.

you know this actually isn't used at all - none of the `client_config` is. i don't think i can remove that wholesale, but i can remove the `authentication` since that's optional. i'll put in dummy values for the `protocol` and `host` to make it clearer that it's not being used.